### ISSUE TITLE
Random Blurry Text Issue

### ISSUE THREAD

**Issue by XandeKK** [XandeKK/TypeX#5]
**Describe the bug**
Some texts appear somewhat blurry, and it's a random behavior. Sometimes, it looks normal on one page, but on other occasions, all texts appear blurry.

**To Reproduce**
As I mentioned, it's a random behavior. One needs to attempt creating and inputting text until it becomes visible.

**Expected behavior**
I would like all texts to maintain good quality.

**Current behavior**
Some texts appear blurry.

**Screenshots**
The upper text is normal, and the lower one is the blurry one.

![01](https://github.com/XandeKK/TypeX/assets/80427458/3ba4df7d-36d8-497d-93fc-f055d7c843c6)

**Desktop (please complete the following information):**
 - OS: Zorin OS
 - Version 16.3

**Additional context**
I'm confident that this issue is caused by the transparent subviewport.

**Comment by XandeKK** [comment]
I went to check if the problem was with the operating system, but it wasn’t. I tested it on Windows 8.1 and this problem also occurs.

But as I stated above, the problem is indeed the Subviewport. By removing the text node in `super_label.tscn` and adjusting the `super_label.gd` code to avoid errors, the text quality became normal, with no blurred text.

But I can’t remove the Subviewport, as it’s necessary for some shaders that will be implemented in the future.

This bug is not problematic, but in a certain way, it can be quite bothersome.
